src/views/EventFilter.js

```diff
@@ -1,24 +1,81 @@
 import React, { Component } from 'react';
+import PropTypes from 'prop-types';
 import {
     StyleSheet,
+    TouchableOpacity,
     Text,
     View
 } from 'react-native';
+import { connect } from 'react-redux';
+import { addEvents, clearEvents } from './../actions/events';
+import { getEvents } from './../api';
+import DateTimePicker from 'react-native-modal-datetime-picker';
+import Moment from 'moment';
 
-export default class EventFilter extends Component {
-    render () {
+
+class EventFilter extends Component {
+    state = {
+        isDateTimePickerVisible: false,
+    };
+
+    _showDateTimePicker = () => this.setState({ isDateTimePickerVisible: true });
+
+    _hideDateTimePicker = () => this.setState({ isDateTimePickerVisible: false });
+
+    _handleDatePicked = (date) => {
+        getEvents(
+            // Set start/end window because backend requires both.
+            { ...this.props.location,
+                start_time: Moment(date).format('YYYY-MM-DD'),
+                end_time: Moment(date).add(1, 'days').format('YYYY-MM-DD') }
+            ).then(
+                res => this.props.addEvents(res),
+                err => console.log(err) || props.addEvents([mockEvent])
```

Comment:
Pretty sure `props` in `props.addEvents` will be undefined here. [ESLint plugin for VSCode will help](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint)

Reply:
It updates the events though 🤔

Reply:
I'll get my linter going.

Reply:
I mean the second one on line 33
`this.props` will work, don't think just `props` will